packages/yourloops/components/menus/user-menu.tsx

```diff
@@ -134,7 +134,7 @@ function UserMenu(): JSX.Element {
         maxWidth={250}
         onClick={event => setAnchorEl(event.currentTarget)}
       >
-        <Box id="user-role-icon" display="flex">
+        <Box data-testid={`${user.role}-icon`} display="flex">
```

Comment:
I suggest keeping `user-role-icon` as a data-testid. I don't see the value in having the role in the data-testid (except making it more difficult to write E2E tests)